tests/linalg_test.py

```diff
@@ -1679,5 +1680,62 @@ def testSchurBatching(self, shape, dtype):
       Ts, Ss = vmap(lax.linalg.schur)(args)
       self.assertAllClose(reconstruct(Ss, Ts), args, atol=1e-4)
 
+  @jtu.sample_product(
+      cshape=[(), (4,), (8,), (3, 7), (0, 5, 1)],
+      cdtype=float_types + complex_types,
+      rshape=[(), (3,), (7,), (2, 1, 4), (19, 0)],
+      rdtype=float_types + complex_types + int_types)
+  def testToeplitzConstrcution(self, rshape, rdtype, cshape, cdtype):
+      if ((rdtype in [np.float64, np.complex128]
```

Comment:
Here and throughout, please use two-space indentations.,

Reply:
done.
What's the style recommendation for hanging indents?
For `_src/scipy/linalg.py`, I followed Google-style, i.e. 2 spaces for normal indents, and 4 spaces for hanging indents.
However, `tests/linalg_test.py` uses 2 spaces also for hanging indents, so I followed that file-local convention only for the test.